osquery/dispatcher/scheduler.cpp

```diff
@@ -109,24 +110,24 @@ void launchQuery(const std::string& name, const ScheduledQuery& query) {
 }
 
 void SchedulerRunner::start() {
+  VLOG(1) << "Starting the schedule runner";
   time_t t = std::time(nullptr);
   struct tm* local = std::localtime(&t);
   unsigned long int i = local->tm_sec;
   for (; (timeout_ == 0) || (i <= timeout_); ++i) {
-    {
-      ConfigDataInstance config;
-      for (const auto& query : config.schedule()) {
-        if (i % query.second.splayed_interval == 0) {
-          launchQuery(query.first, query.second);
-        }
-      }
-    }
+    Config::getInstance().scheduledQueries(
+        ([&i](const std::string& name, const ScheduledQuery& query) {
+          if (i % query.splayed_interval == 0) {
+            launchQuery(name, query);
+          }
+        }));
     // Put the thread into an interruptible sleep without a config instance.
     osquery::interruptableSleep(interval_ * 1000);
   }
 }
 
 Status startScheduler() {
+  VLOG(1) << "Starting scheduler";
```

Comment:
Is a verbose log for "Starting scheduler" and "Starting schedule runner" a bit too verbose? ;)